__tests__/app.test.js

```diff
@@ -83,6 +83,7 @@ describe("/api/reviews/:review_id", () => {
 							category: expect.any(String),
 							created_at: expect.any(String),
 							votes: expect.any(Number),
+							comment_count: expect.any(Number),
```

Comment:
We don't want to change our original test, instead we want to make a new test - in the spirit of TDD, we want to make sure our tests can build upon each other and still work!